crawler.js

```diff
@@ -467,6 +475,20 @@ class Crawler {
       console.warn(e);
     }
   }
+  
+  async createWARCInfo(filename) {
+    const warcVersion = "WARC/1.1";
+    const type = "warcinfo";
+    const info = {
+      "software": `Browsertrix-crawler (via warcio.js 1.4.2)`,
```

Comment:
It would be good to list all the key version here, eg. `Browsertrix-Crawler ${version} (with warcio.js ${warcio_version} pywb ${pywb_version}`).

Browsertrix-Crawler is the easiest, can just read from package.json, warcio.js also from the node_modules/warcio/package.json i think, for pywb, need to check but should be doable as well ideally..